### PR TITLE
Fix for thread order

### LINKED ISSUE #54: Investigate order of sticky thread display
They seem to be in reverse date order?

### PR DESCRIPTION
**Pull request by complacentsee**
This fixes #54 by forcing order on the sticky thread select.

### CODE REVIEW

File: user/showforum.php [hunk 1]
```diff
@@ -173,6 +177,7 @@ function threads($key)
 	$tpl->parse("_row", "row", true);
 
 	$threadshown[$thread['tid']] = 'true';
+  $stickythreads++;
```
Comment (nyetwurk) [inline user/showforum.php:180]: cosmetic, whitespace

File: user/showforum.php [hunk 2]
```diff
@@ -245,6 +260,7 @@ function threads($key)
 	" $ttable.tid <= ? and" .
 	" $ttable.mid >= ? and" .
 	" $ttable.mid <= ? and" .
+  " $ttable.flags NOT LIKE '%STICKY%' and " .   // removing sticky threads from the selection keeps offsets correct if sticky is on page 1
```
Comment (nyetwurk) [inline user/showforum.php:263]: cosmetic, whitespace

File: user/showforum.php [hunk 3]
```diff
@@ -354,4 +375,4 @@ function threads($key)
 print generate_page($forum['name'], $tpl->parse("content", "showforum"), false, $meta_robots);
 
 // vim: sw=2
-?>
+?>
```
Comment (nyetwurk) [inline user/showforum.php:378]: No EOF on EOL, fix your editor :)

**Review by nyetwurk — CHANGES_REQUESTED** [review]
All cosmetic issues with indentation/whitespace

File: user/showforum.php (hunk 3 above)
Comment (complacentsee) [reply user/showforum.php:378]: Fixed issues.

Comment (nyetwurk) [reply user/showforum.php:378]: Still seeing no EOL on EOF ?

Comment (nyetwurk) [reply user/showforum.php:378]: also please rebase/squash thrashing commits :)

Comment (complacentsee) [reply user/showforum.php]: Hopefully I sucked less this time.

**Pull request merged**